README.md

```diff
@@ -18,106 +18,130 @@ Pkg.instantiate()
 
 **Note**: you are strongly encouraged to [open issues](https://github.com/juliaai/DataScienceTutorials.jl/issues/new) on this repository indicating points that are unclear or could be better explained, help us have great tutorials!
 
-## For developers
+## 👩‍💻 For developers
 
 The rest of these instructions assume that you've cloned the package and have `cd` to it.
 
-### Structure
+### 📂 Structure
+The following are the folders relevant to pages on the website:
+```
+├── _literate
+│   ├── data             # has "Data Basics" tutorials
+│   ├── getting-started  # has "Getting Started" tutorials
+│   ├── isl              # has "Introduction to Statistical Learning" tutorials
+│   ├── end-to-end       # has "End-to-End" tutorials
+│   └── advanced         # has "Advanced" tutorials
+├── data                 # This and the four folders below import content from "_literate" to the website 
+├── getting-started
+├── isl
+├── end-to-end
+├── advanced
+├── info                 # has markdown files corresponding to info pages
+├── index.md             # has markdown for the landing page
+├── search.md            # has markdown for the search page
+├── routes.json          # has all the navigation bar data
+├── collapse-script.jl   # script that adds collapsible sections to tutorials
+├── deploy.jl            # deployment script
+├── Manifest.toml
+└── Project.toml         # For the project's environment
+```
+To understand the rest of the structure which could help you change styles with CSS or add interaction with JavaScript read the relevant page on [Franklin's documentation](https://franklinjl.org/workflow/).
 
-All tutorials correspond to a Literate script that's in `_literate/`.
+### 👨🏻‍🔧 Modifying an existing tutorial
 
-### Fixing an existing tutorial
+* Find the corresponding Julia script in `_literate` and fix it in a PR.
+* Ensure it works and renders properly as explained in [this section](#👀-visualise-modifications-locally).
```

Comment:
This internal link (and another like it) do not seem to work, at least in the preview.

Reply:
Well, I followed [this](https://stackoverflow.com/questions/27981247/github-markdown-same-page-link) which claims it works on Github and it does work locally for me. Let's merge and if it doesn't work I will make rightaway make a PR when I see it to replace it with "as in the... section below" or similar.